### PR TITLE
feat: e2e test for copilot

### PR DESCRIPTION
**Pull request by miliPaoletti**

### CODE REVIEW

File: apps/mission/e2e/flow.spec.ts
Comment (sandoche) [inline apps/mission/e2e/flow.spec.ts:1]: Could you please just rename this copilot.spec.ts?

File: apps/mission/e2e/flow.spec.ts [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  test("install Metamask", async ({ page }) => {
+    await page.getByRole("button", { name: /Copilot/i }).click();
+
+    await page
+      .getByRole("button", {
+        name: /Install MetaMask/i,
+      })
+      .click();
+
+    await page.getByText(/Waiting for Metamask Setup/i).isVisible();
```
Comment (sandoche) [inline apps/mission/e2e/flow.spec.ts:39]: Can you use [expect(locator).toBeVisible()](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible) for every isVisible to differentiate the user actions to the tests assertions/expectations?

Comment (sandoche) [reply apps/mission/e2e/flow.spec.ts:39]: Similarly you should also check that when there is no any error that error messages are not visible: [expect(locator).toBeHidden()](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-hidden)

File: apps/mission/e2e/flow.spec.ts [hunk 2]
```diff
@@ -0,0 +1,192 @@
+import { test } from "@playwright/test";
+import {
+  web3Test,
+  web3TestWithoutNetwork,
+} from "playwright-config-custom/helpers";
+
+const BALANCE_ENDPOINT =
+  "*/**/BalanceByDenom/EVMOS/evmos17w0adeg64ky0daxwd2ugyuneellmjgnxpu2u3g/aevmos";
+
+test.beforeEach(async ({ page }) => {
```
Comment (sandoche) [inline apps/mission/e2e/flow.spec.ts:10]: Can you also add describe if it does not make everything crash?

**Review by sandoche — APPROVED** [review]
All good just some tiny changes!

File: apps/mission/e2e/flow.spec.ts (hunk 2 above)
Comment (miliPaoletti) [reply apps/mission/e2e/flow.spec.ts]: where @sandoche ?

Comment (sandoche) [reply apps/mission/e2e/flow.spec.ts]: My bad, it's all good :)

**Review by sandoche — APPROVED** [review]
Perfect

**Pull request merged**